src/storage/mvcc/reader/mod.rs

```diff
@@ -812,4 +819,60 @@ mod tests {
         // `get_txn_commit_info(&key, 15)` stopped at `30_25 PUT`.
         assert_eq!(seek_for_prev_new - seek_for_prev_old, 2);
     }
+
+    #[test]
+    fn test_get_write() {
+        let path = TempDir::new("_test_storage_mvcc_reader_get_write").expect("");
+        let path = path.path().to_str().unwrap();
+        let region = make_region(1, vec![], vec![]);
+        let db = open_db(path, true);
+        let mut engine = RegionEngine::new(Arc::clone(&db), region.clone());
+
+        let (k, v) = (b"k", b"v");
+        let m = Mutation::Put((Key::from_raw(k), v.to_vec()));
+        engine.prewrite(m, k, 1);
+        engine.commit(k, 1, 2);
+
+        engine.rollback(k, 5);
```

Comment:
Don't you need to prewrite something before calling rollback?

Reply:
Yes